internal/federation/server.go

```diff
@@ -119,6 +119,11 @@ func NewServer(t *testing.T, deployment *docker.Deployment, opts ...func(*Server
 	return srv
 }
 
+// Room returns the given room
+func (s *Server) Room(roomID string) *ServerRoom {
```

Comment:
This is not required. You call `MustJoinRoom` in the test which returns the `*ServerRoom` already.